### ISSUE TITLE
Collapsed Hamburger menu does not show up on smaller screens

### ISSUE THREAD

**Issue by peterMuriuki** [opensrp/miecd#25]
![hamburger](https://user-images.githubusercontent.com/28119869/114866094-d3298d80-9dfb-11eb-84b1-fcae46d3d0f8.gif)

**Comment by machariamuguku** [comment]
@p-netm Seeing that the site is web first, I think we should have one responsiveness issue that addresses all issues for smaller devices.

**Comment by peterMuriuki** [comment]
> @p-netm Seeing that the site is web first, I think we should have one responsiveness issue that addresses all issues for smaller devices.
@machariamuguku I agree, though I think its still nice to have this small issues as they come up, for ease of work division and also will  lessen the scope of work per pr.

**Comment by machariamuguku** [comment]
> > @p-netm Seeing that the site is web first, I think we should have one responsiveness issue that addresses all issues for smaller devices.
> > @machariamuguku I agree, though I think its still nice to have this small issues as they come up, for ease of work division and also will  lessen the scope of work per pr.

I agree, but since responsiveness is out of scope for this refactor, we can have one huge issue with bullet points as they arise (something like this #18 ) to be broken down to individual issues when time to refactor comes. This will help avoid noise on the issues at hand
